src/main/java/wolox/training/models/Book.java

```diff
@@ -50,6 +52,9 @@
 	@Column(nullable = false)
 	private String isbn;
 
+	@ManyToMany(mappedBy = "books")
+	List<User> users;
```

Comment:
add `JsonIgnore` to avoid circular references

Reply:
I recognized this issue when I was writing the Swagger docs. I've come with a solution, better than @JsonIgnore, in the swagger-implementation PR